app/dts/input/processors/behaviors.dtsi

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2024 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+    zip_button_behaviors: zip_button_behaviors {
+        compatible = "zmk,input-processor-behaviors";
+        #input-processor-cells = <0>;
+        codes =    <INPUT_BTN_0 INPUT_BTN_1 INPUT_BTN_2>;
+        bindings = <&none       &none       &none>;
```

Comment:
Also, is it better to map all five keys?

Reply:
Me only concern with using `&mkp` be default is the potential for confusion for folks thinking they need to use this to get buttons to send HID events, and also in theory the risk of an infinite loop if you applied this to the `&mkp_input_listener`.

Regarding all 5 keys... I considered this. I don't think the 4th and 5th keys get generated nearly as often, and wanted to avoid folks needing to have unneeded placeholders, and more flash space used, for those infrequently used buttons. Folks can always override the `codes` property if they really need to.